src/components/SearchBar/SearchBar.test.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import renderer from 'react-test-renderer';
+
+import SearchBar from './index';
+
+import history from '../../data/history';
+
+const props = {
+  history,
+};
+
+function search(input, searchQuery) {
+  input.simulate('change', { target: { value: searchQuery } });
+  input.simulate('keypress', { key: 'Enter' });
+}
+
+describe('<SearchBar />', () => {
+  describe('renders', () => {
+    const wrapper = mount(<SearchBar {...props} />);
+    expect(wrapper.exists()).toBe(true);
+
+    it('correctly', () => {
+      const tree = renderer
+        .create(<SearchBar {...props} />)
+        .toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+
+    it('with empty search input', () => {
```

Comment:
Should not this be covered under snapshot test above?

Reply:
agreed :+1: